internal/http/services/owncloud/ocdav/locks.go

```diff
@@ -566,6 +579,13 @@ func writeLockInfo(w io.Writer, token string, ld LockDetails) (int, error) {
 	if href != "" {
 		lockdiscovery.WriteString(fmt.Sprintf("  <d:lockroot><d:href>%s</d:href></d:lockroot>\n", prop.Escape(href)))
 	}
+	if ld.OwnerName != "" {
+		lockdiscovery.WriteString(fmt.Sprintf("  <d:ownername>%s</d:ownername>\n", prop.Escape(ld.OwnerName)))
+	}
+	if !ld.Locktime.IsZero() {
+		lockdiscovery.WriteString(fmt.Sprintf("  <d:locktime>%s</d:locktime>\n", prop.Escape(ld.Locktime.Format(time.RFC3339))))
```

Comment:
You cannot use the RFC DAV namespace when extending the xml. For that we need to use the owncloud namespace. Since we are writing the xml ourself, I think we can hardcode `oc`.
```suggestion
		lockdiscovery.WriteString(fmt.Sprintf("  <oc:ownername>%s</oc:ownername>\n", prop.Escape(ld.OwnerName)))
	}
	if !ld.Locktime.IsZero() {
		lockdiscovery.WriteString(fmt.Sprintf("  <oc:locktime>%s</oc:locktime>\n", prop.Escape(ld.Locktime.Format(time.RFC3339))))
```

Reply:
Good point. Changed :+1: